packages/blocks/src/_common/components/hover/controller.ts

```diff
@@ -131,7 +133,10 @@ export class HoverController implements ReactiveController {
 
       const transition = this._hoverOptions.transition;
       if (transition) {
-        Object.assign(this._portal.style, transition.in);
+        setTimeout(() => {
+          assertExists(this._portal);
+          Object.assign(this._portal.style, transition.in);
+        });
```

Comment:
Revert the `setTimeout` to prevent sometimes the tooltip not displayed.

Reply:
https://github.com/toeverything/blocksuite/assets/123532141/0d092e03-3318-4469-a328-9d4f354952de

I guess the problem arises because abortController.abort() is not being triggered in the transition end event due to rapid mouse movements. Which worked when I removed the transition in my PR.

Reply:
This is becase the `transitionend` listener was remove since `transitioncancel` event happend, which make the `abortController` never abort, and  the opacity of portal is always zero.

https://developer.mozilla.org/en-US/docs/Web/API/Element/transitionend_event

![image](https://github.com/toeverything/blocksuite/assets/20479050/eccc5d4e-9d8e-4afa-a1b6-7b352ca24e75)